text/0117-typo-tolerance-settings-api.md

```diff
@@ -0,0 +1,202 @@
+- Title: Typo Tolerance Settings API
+- Start Date: 2022-02-22
+
+# Typo Tolerance API
+
+## 1. Summary
+
+This specification describes the customizable options for the typo tolerance settings.
+
+## 2. Motivation
+
+Tolerance to typos is critical when building an enjoyable search experience. Meilisearch ranks typo-free matched documents above others based on the default settings that we believe are best for most users.
+
+Meilisearch can adapt to different use-cases by providing customization options for the typo tolerance feature.
+
+## 3. Functional Specification
+
+### 3.1. `typoTolerance` API resource definition
+
+| Field                                            | Type            | Required |
+|--------------------------------------------------|-----------------|----------|
+| [enabled](#311-enabled)                          | Boolean         | False    |
+| [disableOnAttributes](#312-disableonattributes)  | Array of String | False    |
+| [disableOnWords](#313-disableonwords)            | Array of String | False    |
+| [minWordSizeFor1Typo](#314-minwordsizefor1typo)  | Integer         | False    |
+| [minWordSizeFor2Typos](#315-minwordsizefor2typos)| Integer         | False    |
+
+#### 3.1.1. `enabled`
+
+- Type: Boolean
+- Required: False
+- Default: `true`
+
+Whether the typo tolerance feature is enabled.
+
+##### 3.1.1.1. Impacts on the `typo` ranking rule
+
+The presence of `typo` in the ranking rules setting does not influence the activation/deactivation of the typo tolerance feature.
```

Comment:
Im not sure I understood this sentence correctly :s

Reply:
Basically, if the `typo` ranking rule is removed from the setting, the engine will still be able to match documents containing typo errors against the query term. It will just not rely on the number of typos to rank the documents.

It it clearer this way?

Reply:
So it would still be possible to receive 
`helol` instead of `hello` even when removing `typo` from the ranking rules? But it will be in any random order?

Reply:
Yes, the order will of course be defined by all other ranking rules.

Reply:
ah I see! Thanks :)